model/map/src/main/java/org/keycloak/models/map/storage/chm/MapFieldPredicates.java

```diff
@@ -613,4 +618,8 @@ public static <K, V extends AbstractEntity, M> Comparator<V> getComparator(Strea
     public static <K, V extends AbstractEntity, M> Map<SearchableModelField<? super M>, UpdatePredicatesFunc<K, V, M>> getPredicates(Class<M> clazz) {
         return PREDICATES.get(clazz);
     }
+
+    public static boolean isPrimaryKey(SearchableModelField<?> field) {
```

Comment:
Primary key is not relevant in logical layer, so the name should be amended

```suggestion
    public static boolean isIdField(SearchableModelField<?> field) {
```

Reply:
I removed the method since it is no longer necessary with the method [getSingleRestrictionArgument](https://github.com/keycloak/keycloak/blob/3be2775f9e72845a1f4c178d8c197cc17488b7aa/model/map/src/main/java/org/keycloak/models/map/storage/criteria/DescriptiveModelCriteria.java#L181)